packages/commons/src/sqs/index.ts

```diff
@@ -229,4 +229,4 @@ export {
   queueManagerSendError,
   queueManagerReceiveError,
 };
-export type { SQSClientConfig, Message };
+export type { SQSClientConfig };
```

Comment:
I would also remove other re-exports from `"@aws-sdk/client-sqs"` - `SQSClientConfig` and `SQSClient`. They can just be imported directly from `"@aws-sdk/client-sqs"` where needed